packages/nextjs/src/utils/isBuild.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Decide if the currently running process is part of the build phase or happening at runtime.
+ */
+export function isBuild(): boolean {
```

Comment:
Totally optional: Since we extracted this as a function, we could add a test for it. WDYT?

Reply:
Hmmm. I'm trying to think how you'd do that. Any ideas?

Reply:
Hmm that's a good question 😅 I suppose we can't just add an item to `proces.argv` before calling the function? Just found this for process.env: https://stackoverflow.com/a/48042799

Reply:
Oh, yeah, I guess that's true. I was overcomplicating things and thinking about how to actually run an executable with the argument `build` and spawn child processes and all. 🤦🏻‍♀️ 

Anyway, done.